src/sipsess/sess.c

```diff
@@ -372,6 +372,21 @@ bool sipsess_ack_pending(const struct sipsess *sess)
 }
 
 
+/**
+ * Compares the sessions SIP message with given SIP message
+ *
+ * @param sess  SIP Session
+ * @param msg   SIP Message
+ * @return True if msg matches with sess->msg
+ */
+bool sipsess_msg_equal(const struct sipsess *sess, const struct sip_msg *msg)
```

Comment:
it would also be more flexible to add a getter for sess->msg and let the application
do the comparison